src/org/jgroups/protocols/SSL_KEY_EXCHANGE.java

```diff
@@ -82,6 +134,16 @@ public interface SessionVerifier {
     @Property(description="Password to access the keystore",exposeAsManagedAttribute=false)
     protected String          keystore_password="changeit";
 
+    @Property(description="Location of the truststore. Defaults to null, which will use the keystore as a truststore.")
+    protected String          truststore_name;
+
+    @Property(description="The type of the truststore. " +
+      "Types are listed in http://docs.oracle.com/javase/8/docs/technotes/tools/unix/keytool.html")
+    protected String          truststore_type="JKS";
```

Comment:
We should probably change the default to PKCS12 as it is also the JDK default now

Reply:
Fixed.